include/grpcpp/server_builder.h

```diff
@@ -210,15 +210,46 @@ class ServerBuilder {
   /// doc/workarounds.md.
   ServerBuilder& EnableWorkaround(grpc_workaround_list id);
 
- private:
-  friend class ::grpc::testing::ServerBuilderPluginTest;
-
+ protected:
   struct Port {
```

Comment:
Let's also add experimental tags on these structs that have been moved from private to protected.